hg-fast-export.py

```diff
@@ -123,19 +123,32 @@ def get_author(logmessage,committer,authors):
       return r
   return committer
 
-def export_file_contents(ctx,manifest,files,hgtags,encoding=''):
+def export_file_contents(ctx,manifest,files,hgtags,encoding='',filter_contents=None):
   count=0
   max=len(files)
   for file in files:
     # Skip .hgtags files. They only get us in trouble.
     if not hgtags and file == ".hgtags":
       sys.stderr.write('Skip %s\n' % (file))
       continue
-    d=ctx.filectx(file).data()
     if encoding:
       filename=file.decode(encoding).encode('utf8')
     else:
       filename=file
+    f=ctx.filectx(file)
```

Comment:
`f` what is it? Call it `file_info` so I that I don't mix it up with all the other file related variables.